docs/websites/README.md

```diff
@@ -54,10 +60,13 @@ The link file also appears locally.
 
 After this, you can access the website at the following ports:
 
-| Website      |          URL          |
-|--------------|-----------------------|
-|help.keyman   | http://localhost:8055 |
-|keymanweb.com | http://localhost:8057 |
+| Website        |          URL          |  with website-local-proxy running |
+|----------------|-----------------------|-----------------------------------|
+| keyman.com     | http://localhost:8053 | http://keyman.com.localhost       |
+| s.keyman.com   | http://localhost:8054 | http://s.keyman.com.localhost     |
+| help.keyman    | http://localhost:8055 | http://help.keyman.com.localhost  |
+| keymanweb.com  | http://localhost:8057 | http://keymanweb.com.localhost    |
```

Comment:
does web.keyman.com.localhost also work?

Reply:
err, confusingly that link gets me api.keyman.  I'll have to investigate why...

Reply:
hmm, that does sound like it needs to be fixed!

Reply:
Added in keymanapp/website-local-proxy#6

Reply:
```suggestion
| keymanweb.com  | http://localhost:8057 | http://keymanweb.com.localhost    |
|                                  |                                       | http://web.keyman.com.localhost   |